rpgatta_msgs/MotionPlan plan # milliseconds
---
# Success if empty
string error
